cp3-networkpolicy/egress/license-service-reporter/ibm-license-service-reporter-operator-egress.yaml

```diff
@@ -1,17 +1,15 @@
 apiVersion: networking.k8s.io/v1
```

Comment:
please do not rename the filename - keep the original filename

Reply:
file name doesn't matter. 
.sh running everything from folder
will leave "ibm-license-service-reporter-operator-egress.yaml" - more readable for what and what is in the content

Reply:
I don't agree - for the upgrade scenario, the old networkpolicy will not be updated

Reply:
changed in other comment